lib/table.c

```diff
@@ -33,12 +33,36 @@ DEFINE_MTYPE(LIB, ROUTE_NODE, "Route node")
 
 static void route_table_free(struct route_table *);
 
+static int route_table_opaque_hash_cmp(const void *a, const void *b)
```

Comment:
This imo is not the way to do this.  A bit of special case code for flowspec instead of abstracting it into the prefix?  Make flowspec prefix types use the normal prefix functions and the entirety of these changes go away

Reply:
Additionally is a table the correct data store for flowspec data.  The point of a table is that I can store prefixes that are alike and if I want to match a better prefix I walk up the tree.  Do you even need to do this with a flowspec prefix?  What is the advantage to using this data structure?

Reply:
Upon further consideration, bgp forces us to use a table for flowspec, but if we make the changes to fold the prefix_XXX code into the normal prefix function flow, then all the table changes become unnecessary, right?